package.json

```diff
@@ -43,7 +43,7 @@
     "ready-state": "^2.0.5"
   },
   "peerDependencies": {
-    "react": "^16.9.0"
+    "react": ">=16.9.0"
```

Comment:
**suggestion**: would it be okay if this was a bit more defensive and doesn't accept a future React 18+ just yet?

```suggestion
    "react": ">=16.9.0 <18.0.0"
```